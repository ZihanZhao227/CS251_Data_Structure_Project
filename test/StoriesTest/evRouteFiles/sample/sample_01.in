10
0 0 0 CITY StartCity
1 50 0 CHARGING_STATION ChargingStation1
2 100 0 CHARGING_STATION ChargingStation2
3 25 25 RESTAURANT Restaurant1
4 75 25 TOURIST_ATTRACTION Museum1
5 150 0 CHARGING_STATION ChargingStation3
6 200 0 CHARGING_STATION ChargingStation4
7 125 50 RESTAURANT Restaurant2
8 175 25 TOURIST_ATTRACTION Park1
9 250 0 CITY DestinationCity