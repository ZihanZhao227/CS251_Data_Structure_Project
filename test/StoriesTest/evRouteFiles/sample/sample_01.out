0 9 75.0 250.000
0 1 2 5 6 9